aspnet/tutorials/first-mvc-app/adding-view.rst

```diff
@@ -131,10 +131,10 @@ Save your changes and browse to the following URL:
 
 \http://localhost:xxxx/HelloWorld/Welcome?name=Rick&numtimes=4
 
-Data is taken from the URL and passed to the controller using the `model binder <http://docs.asp.net/projects/mvc/en/latest/models/index.html>`__. The controller packages the data into a ``ViewData`` dictionary and passes that object to the view. The view then renders the data as HTML to the browser.
+Data is taken from the URL and passed to the controller using the `model binder </mvc/models>`__. The controller packages the data into a ``ViewData`` dictionary and passes that object to the view. The view then renders the data as HTML to the browser.
```

Comment:
@Rick-Anderson \ @danroth27 - I wanted to link to https://docs.asp.net/en/dev/mvc/models/index.html instead of pointing to the latest. Does this link look correct to you?